### PR TITLE
#164 when you were deleting extra blocks that were trashed mesh would…

### PR DESCRIPTION
**Pull request by aaronware**
… get a bit too greedy.

### CODE REVIEW

File: assets/js/admin-mesh-blocks.js [hunk 1]
```diff
@@ -334,8 +341,31 @@ mesh.blocks = function ( $ ) {
                         console.log( e );
                     }
 
+                    console.log( self.get_block_cache( editor_id ) );
```
Comment (nate-allen) [inline assets/js/admin-mesh-blocks.js]: Remove console.log

File: assets/js/admin-mesh-blocks.js [hunk 2]
```diff
@@ -334,8 +341,31 @@ mesh.blocks = function ( $ ) {
...
+
+                    /*
+                     * Cache
+                     */
+                    if ( typeof tinymce !== 'undefined' ) {
+                        var editor = tinymce.get( editor_id ),
+                            cached_block_content = self.get_block_cache(editor_id);
+
+                        console.log( editor_id );
+                        console.log( editor );
```
Comment (nate-allen) [inline assets/js/admin-mesh-blocks.js]: Remove these 2 console.logs

File: assets/js/admin-mesh.js [hunk 3]
```diff
@@ -598,8 +605,31 @@ mesh.blocks = function ( $ ) {
                         console.log( e );
                     }
 
+                    console.log( self.get_block_cache( editor_id ) );
```
Comment (nate-allen) [inline assets/js/admin-mesh.js]: Remove console.log

File: assets/js/admin-mesh.js [hunk 4]
```diff
@@ -598,8 +605,31 @@ mesh.blocks = function ( $ ) {
...
+
+                    /*
+                     * Cache
+                     */
+                    if ( typeof tinymce !== 'undefined' ) {
+                        var editor = tinymce.get( editor_id ),
+                            cached_block_content = self.get_block_cache(editor_id);
+
+                        console.log( editor_id );
+                        console.log( editor );
```
Comment (nate-allen) [inline assets/js/admin-mesh.js]: Remove these console.logs

**Comment by aaronware** [comment]
@nate-allen I removed the `console.log` items. Can you OK the review?

**Pull request merged**